scRNA-seq-advanced/02-celltype_assignment.Rmd

```diff
@@ -0,0 +1,465 @@
+---
+title: "Assigning Cell Types"
+author: Data Lab for ALSF
+date: 2022
+output:
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+## Objectives
+
+This notebook will demonstrate how to:
+
+- 
+
+---
+
+In this notebook, we will attempt to assign cell types to each of the cells in a dataset, using some of the automated tools that are available within the Bioconductor universe.
+
+
+Much of the material in this notebook is directly inspired by, and draws heavily on, material presented in the book [_Orchestrating Single Cell Analysis with Bioconductor_](http://bioconductor.org/books/3.16/OSCA/). 
+
+The data we will use for this notebook is derived from a 10x Genomics Dataset: https://software.10xgenomics.com/single-cell-gene-expression/datasets/6.0.0/10k_PBMCs_TotalSeq_B_3p. 
+The data here have already been filtered, normalized, and had dimension reductions calculated.
+
+In addition to gene expression data, this data set includes data from antibody-derived tags (ADTs) which are used to mark cell-surface proteins. 
+
+## Set Up
+
+To start, we will load some of the libraries we will need later, and set a random number seed for reproducibility.
+
+```{r setup}
+# Load libraries
+library(magrittr) # the pipe (%>%) operator
+library(ggplot2) # plotting functions
+library(SingleCellExperiment) # our main class for storing Single Cell data
+
+
+# Setting the seed for reproducibility
+set.seed(12345)
+```
+
+
+### Input and output
+
+```{r}
```

Comment:
General note to include chunk names (unless you are planning this for a future PR).

Reply:
Yep, future problems.